packages/story-editor/src/components/panels/design/animation/effectChooserDropdown/effectChooserDropdown.js

```diff
@@ -83,15 +79,9 @@ const EffectChooserDropdown = forwardRef(function EffectChooserDropdown(
     ? backgroundEffectOptions
     : foregroundEffectOptions;
 
-  // remove experiments if needed
   const availableAnimationOptions = useMemo(
-    () =>
-      enableExperimentalAnimationEffects
-        ? Object.values(animationOptionsObject)
-        : Object.values(animationOptionsObject).filter(({ value }) => {
-            return experimentalEffects.indexOf(value) === -1;
-          }),
-    [animationOptionsObject, enableExperimentalAnimationEffects]
+    () => Object.values(animationOptionsObject),
```

Comment:
Probably a bit overkill to use `useMemo` now

Reply:
Yeah, I was considering removing it. I've done so in f628d0da05.